workflow/src/main/java/com/google/android/fhir/workflow/FhirOperator.kt

```diff
@@ -61,23 +60,25 @@ import org.opencds.cqf.cql.evaluator.plandefinition.r4.PlanDefinitionProcessor
 // This is a heavy object to load. Avoid having to create a new one.
 object cachedR4FhirModelResolver : R4FhirModelResolver(FhirContext.forR4Cached())
```

Comment:
Class name should start with an uppercase letter. Could you fix that please?
```suggestion
object CachedR4FhirModelResolver : R4FhirModelResolver(FhirContext.forR4Cached())
```